R/oc_forward.R

```diff
@@ -31,58 +34,89 @@
 #'   addresses in the \code{formatted} field of the results are abbreviated
 #'   (e.g. "Main St." instead of "Main Street").
-#' @param add_request A logical vector (default \code{FALSE}) indicating whether
+#' @param add_request Logical vector (default \code{FALSE}) indicating whether
 #'   the request is returned again with the results. If the \code{return} value
 #'   is a \code{df_list}, the query text is added as a column to the results.
 #'   \code{json_list} results will contain all request parameters, including the
-#'   API key used! For \code{geojson_list} this is currently ignored by
-#'   OpenCage.
+#'   API key used! This is currently ignored by OpenCage if return value is
+#'   \code{geojson_list}.
 #' @param ... Ignored.
 #'
-#' @return \code{oc_forward} returns, depending on the \code{return} parameter,
-#'   a list with either
+#' @return Depending on the \code{return} argument, \code{oc_forward} returns a
+#'   list with either
 #'   \itemize{
 #'   \item the results as tibbles (\code{"df_list"}, the default),
 #'   \item the results as JSON lists (\code{"json_list"}),
 #'   \item the results as GeoJSON lists (\code{"geojson_list"}), or
 #'   \item the URL of the OpenCage API call for debugging purposes
 #'   (\code{"url_only"}).
 #'   }
-#'   \code{oc_forward_df} returns a tibble.
 #'
-#' @seealso \code{\link{oc_reverse}} for reverse geocoding, and the
-#'   \href{https://opencagedata.com/api}{OpenCage API documentation} for more
-#'   information about the API.
+#' @seealso \code{\link{oc_forward_df}} for inputs as a data frame, or
+#'   \code{\link{oc_reverse}} and \code{\link{oc_reverse_df}} for reverse
+#'   geocoding. For more information about the API and the various parameters,
+#'   see the \href{https://opencagedata.com/api}{OpenCage API documentation}.
 #'
 #' @export
 #'
 #' @examples
 #' \dontrun{
-#' oc_forward(placename = "Sarzeau")
-#' oc_forward(placename = "Islington, London")
+#' # Geocode a single location, an address in this case
 #' oc_forward(placename = "Triererstr 15,
 #'                         Weimar 99423,
 #'                         Deutschland")
+#'
+#' # Geocode multiple locations
+#' locations <- c("Nantes", "Hamburg", "Los Angeles")
+#' oc_forward(placename = locations)
+#'
+#' # Use bounding box to help return accurate results
+#' # for each placename
+#' bounds <- oc_bbox(xmin = c(-2, 9, -119),
+#'                   ymin = c(47, 53, 34),
+#'                   xmax = c(0, 10, -117),
+#'                   ymax = c(48, 54, 35))
+#' oc_forward(placename = locations, bounds = bounds)
+#'
+#' # Another way to help specify the desired results
+#' # is with country codes.
+#' oc_forward(placename = locations,
+#'            countrycode = c("ca", "us", "co"))
+#'
+#' # With multiple countrycodes per placename
+#' oc_forward(placename = locations,
+#'            countrycode = list(c("fr", "ca") , c("de", "us"), c("us", "co"))
+#'            )
+#'
+#' # Favor results in a specified language
```

Comment:
🤔 Do we write our docs in en-GB or en-US? (I don't care too much, but it matters for spell-checking with {[spelling](https://docs.ropensci.org/spelling)}).

Could this be misinterpreted as "Favour (i.e. select) results that have a french name" as opposed to "Return the results you find, and if they have a french name, show me that"? How about "Return results in the preferred language if possible".

Reply:
I vote US English 🇺🇸. But I guess I am biased. I agree on your point about the phrasing. I changed it to "Return results in a preferred language if possible"